File: src/main/java/baboon/industry/gui/plugins/WailaPlugin.java
package baboon.industry.gui.plugins;

import org.slf4j.Logger;
import toufoumaster.btwaila.BTWaila;
import toufoumaster.btwaila.BTWailaCustomTootltipPlugin;

public class WailaPlugin implements BTWailaCustomTootltipPlugin {

    @Override
    public void initializePlugin(Logger logger) {
        BTWaila.LOGGER.info("Initialized tooltips for Industry2.");
    }
}

File: src/main/kotlin/turniplabs/industry/gui/plugins/WailaPlugin.kt
package turniplabs.industry.gui.plugins

import org.slf4j.Logger
import toufoumaster.btwaila.BTWaila
import toufoumaster.btwaila.BTWailaCustomTootltipPlugin

class WailaPlugin : BTWailaCustomTootltipPlugin {

    override fun initializePlugin(logger: Logger?) {
        BTWaila.LOGGER.info("Adding tooltips for Industry2.")

        GeneratorTooltip().addTooltip()
        EnergyConductorTooltip().addTooltip()
    }
}